ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -347,6 +347,55 @@ def test_cumsum(
     )
 
 
+@handle_test(
+    fn_tree="functional.ivy.cummax",
+    dtype_x_axis_castable=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=1,
+        max_num_dims=2,
```

Comment:
Why are we restricting the `max_num_dims` to 2?

Reply:
Okay I'll look into other functions such as cumprod and cumsum . I'll give the same max dimensions and test it.